extensions/sld/include/powsybl/iidm/extensions/sld/ConnectablePosition.hpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_EXTENSIONS_SLD_CONNECTABLEPOSITION_HPP
+#define POWSYBL_IIDM_EXTENSIONS_SLD_CONNECTABLEPOSITION_HPP
+
+#include <boost/optional.hpp>
+
+#include <powsybl/iidm/Extension.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+class Connectable;
+
+namespace extensions {
+
+namespace sld {
+
+class ConnectablePosition : public Extension {
+public:
+    enum class Direction : unsigned char {
+        TOP,
+        BOTTOM,
+        UNDEFINED
+    };
+
+    class Feeder {
+    public:
+        Feeder(const std::string& name, unsigned long order, const Direction& direction);
+
+        const Direction& getDirection() const;
+
+        const std::string& getName() const;
+
+        unsigned long getOrder() const;
+
+        Feeder& setDirection(const Direction& direction);
+
+        Feeder& setName(const std::string& name);
+
+        Feeder& setOrder(unsigned long order);
+
+    private:
+        std::string m_name;
+
+        unsigned long m_order;
+
+        Direction m_direction;
+    };
+
+    using OptionalFeeder = boost::optional<Feeder>;
```

Comment:
To be discussed: in the other classes, we use the `stdcxx::reference_wrapper` that allow empty references. Maybe we should use the same design in the `ConnectablePosition` class, but an `OptionalFeeder` in the adder?

Reply:
I know what you mean, but the problem with usage of reference_wrapper is about ownership of the feeder. 
However, I agree that getters should return stdcxx::reference_wrapper.

Reply:
The attributes are declared as `OptionalFeeder` but the getters should return a `stdcxx::Reference` or `stdcxx::CReference`